can/interfaces/kvaser/canlib.py

```diff
@@ -471,6 +471,7 @@ def __init__(
         log.info("Found %d available channels", num_channels)
         for idx in range(num_channels):
             channel_info = get_channel_info(idx)
+            channel_info = f"{channel_info["device_name"]}, S/N {channel_info["serial"]} (#{channel_info["dongle channel"]})"
```

Comment:
Double quotes inside of double quotes is a syntax error in older Python versions.

Reply:
Sorry, I just started using Python last year with the latest version and I'm unaware of these retrocompatibility issues. Can I change it using single quotes for the f-string but maitain the dict indexing inside it with double quotes? Thanks in advance